dmoj/executors/SCALA.py

```diff
@@ -20,6 +20,7 @@ class Executor(JavaExecutor):
         ExactFile('/bin/bash'),
         RecursiveDir('/etc/alternatives'),
     ]
+    compiler_syscalls = ['mincore']
```

Comment:
Why did this change?

Reply:
SCALA takes a different path under landlock last I checked, and that path requires `mincore`. I believe it had to do with how landlock deals with `/proc/<pid>`.

We decided at some point to simply allow the call.

Reply:
Ack, thanks for explaining. Could we split this out in its own commit?

Reply:
Eh, I guess it kind of is already. Fine to leave it as-is.